Question1_BinToOct.java

```diff
@@ -0,0 +1,59 @@
+/**
+ * This class is designed to convert a given binary no to equivalent octal value
+ * @author Gaurav Tak
+ *
+ */
+public class Question1_BinToOct {
+	/**
+	 * 
+	 * @param n is assumed a sequence of bits
+	 * @return octal value for binary n
+	 */
+	public int convertBinaryToOctal(int n) {
+		int triad=0; //holds group of consecutive three bits
+		String octalValue="";
+		while(n>0) {
+			triad=n%1000;
+			octalValue=getOctalEquivalent(triad)+octalValue;
+			n=n/1000;
+		}
+		return Integer.parseInt(octalValue);
```

Comment:
Better to use 
return StringUtils.isBlank(myString) ? 0 : Integer.parseInt(myString);
else it will fail for 0